tests/integration/devfile/cmd_devfile_create_test.go

```diff
@@ -149,6 +149,14 @@ var _ = Describe("odo devfile create command tests", func() {
 		})
 	})
 
+	Context("When executing odo create with component with no devBuild command", func() {
```

Comment:
Can we add a test case that checks if it lists a devfile without devrun? Is that possible or too much to do ie change to a non default registry and such?

Reply:
I would leave it out for now, as it requires setting up a separate devfile registry just for that step.

Reply:
I mean, with no `devbuild`

Reply:
Would that be the quarkus devfile in this case? (Which we're testing)